package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "lev-api",
-  "version": "0.13.0",
+  "version": "0.14.0",
```

Comment:
We'll need to bump the spec as well.

Reply:
Already done and PR raised:

https://github.com/UKHomeOffice/lev-api-spec/pull/30

Reply:
Although as a draft until this PR is ready